boards/nxp/frdm_mcxn947/doc/index.rst

```diff
@@ -0,0 +1,202 @@
+.. _frdm_mcxn947:
+
+NXP FRDM-MCXN947
+################
+
+Overview
+********
+
+FRDM-MCXN947 are compact and scalable development boards for rapid prototyping of
+MCX N94 and N54 MCUs. They offer industry standard headers for easy access to the
+MCU�s I/Os, integrated open-standard serial interfaces, external flash memory and
+an on-board MCU-Link debugger. MCX N Series are high-performance, low-power
+microcontrollers with intelligent peripherals and accelerators providing multi-tasking
+capabilities and performance efficiency.
+
+.. image:: frdm_mcxn947.jpg
+   :align: center
+   :alt: FRDM-MCXN947
+
+Hardware
+********
+
+- MCX-N947 Dual Arm Cortex-M33 microcontroller running at 150 MHz
+- 2MB dual-bank on chip Flash
+- 512 KB RAM
+- External Quad SPI flash over FlexSPI
+- USB high-speed (Host/Device) with on-chip HS PHY. HS USB Type-C connectors
+- 10x LP Flexcomms each supporting SPI, I2C, UART
+- 2x FlexCAN with FD, 2x I3Cs, 2x SAI
+- 1x Ethernet with QoS
+- On-board MCU-Link debugger with CMSIS-DAP
+- Arduino Header, FlexIO/LCD Header, SmartDMA/Camera Header, mikroBUS
+
+For more information about the MCX-N947 SoC and FRDM-MCXN947 board, see:
+
+- `MCX-N947 SoC Website`_
+- `MCX-N947 Datasheet`_
+- `MCX-N947 Reference Manual`_
+- `FRDM-MCXN947 Website`_
+- `FRDM-MCXN947 User Guide`_
+- `FRDM-MCXN947 Schematics`_
+
+Supported Features
+==================
+
+The FRDM-MCXN947 board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | clock_control                       |
++-----------+------------+-------------------------------------+
+
+Targets available
+==================
+
+The default configuration file
+``boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0_defconfig``
```

Comment:
```suggestion
:zephyr_file:`boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0_defconfig`
```

Reply:
Thanks, I have changed it.

Reply:
Not resolved

Reply:
I had pushed the change. PR has been updated with this change.